packages/wellknown-did-verifier/README.md

```diff
@@ -0,0 +1,95 @@
+<!--suppress HtmlDeprecatedAttribute -->
+<h1 align="center">
+  <br>
+  <a href="https://www.sphereon.com"><img src="https://sphereon.com/content/themes/sphereon/assets/img/logo.svg" alt="Sphereon" width="400"></a>
+  <br>Well-Known DID Verifier (Typescript) 
+  <br>
+</h1>
+
+---
+
+**Warning: This package still is in very early development. Breaking changes without notice will happen at this point!**
+
+---
+
+A `Sphereon SSI-SDK` plugin to verify relationships between the controller of an origin and a DID and to verify DID configuration resources.
+
+## Available functions
+* registerSignatureValidation
+* removeSignatureValidation
+* verifyDomainLinkage
+* verifyDidConfigurationResource
+
+## Usage
+
+### Adding the plugin to an agent:
+
+```typescript
+import { 
+  IWellKnownDidVerifier, 
+  WellKnownDidVerifier 
+} from '@sphereon/ssi-sdk-wellknown-did-verifier';
+
+const agent = createAgent<IWellKnownDidVerifier>({
+  plugins: [
+    new WellKnownDidVerifier({
+      signatureVerifications: {'verified': () => Promise.resolve({ verified: true })},
+      onlyVerifyServiceDids: true
+    }),
+  ],
+})
+```
+
+### Register signature verification callback:
+
```

Comment:
Could we add a line what this is actually doing?

Reply:
added description